src/main/java/org/noteskeeper/server/IGreetingService.java

```diff
@@ -0,0 +1,5 @@
+package org.noteskeeper.server;
+
+public interface IGreetingService {
```

Comment:
Plural implies collection of items which is the case with GreetingService. 'Impl' suffix sounds good to me though.